Cargo.toml

```diff
@@ -14,5 +14,13 @@ readme = "README.md"
 [workspace]
 members = ["coin_cbc_sys"]
 
+[features]
+default = ["singlethread-cbc"]
+# The `singlethread-cbc` feature MUST be activated if this crate is going to be used with
+# a cbc dynamic library that has not been explicitly compiled with `-DCBC_THRED_SAFE`.
+# For more information, see https://github.com/KardinalAI/coin_cbc/issues/9
```

Comment:
Please add this documentation somewhere more visible, as in the README.md and the top level documentation at the head of lib.rs

Reply:
@TeXitoi : done in 353c5e3